main.py

```diff
@@ -8,7 +8,40 @@
 from steps import STEPS
 from db import DB, DBs
 import typer
+from dotenv import load_dotenv
 
+load_dotenv()
+
+api_key = os.getenv("OPENAI_API_KEY") 
+# Check if OPENAI_API_KEY is defined
+if api_key == None:
+    # Raise an error if OPENAI_API_KEY is not defined
+    raise ValueError("OPENAI_API_KEY is not defined")
+else:
+    # Set the OPENAI_API_KEY to the api_key variable
+    openai.api_key = api_key
+
+# Get the OPENAI_API_BASE
+api_base = os.getenv("OPENAI_API_BASE")
+# Check if OPENAI_API_BASE is defined
+if api_base != None:   
+    # Set the OPENAI_API_BASE to the api_base variable
+    openai.api_base = api_base
+
+# Get the list of valid GPT models
+valid_gpt_modles = [
```

Comment:
Nothing critical but there is a typo here! 
`valid_gpt_models` not `valid_gpt_modles`

Reply:
Support for gpt-3.5-turbo-16k should be added as it is the new 16k model

Reply:
https://github.com/AntonOsika/gpt-engineer/pull/29/commits/b5ed9ecaf658848f7dd3a6a429e8eb63f2fd5850

Model validation is removed, probably better to just let users figure out which model they want to use